pkg/util/common.go

```diff
@@ -655,7 +655,11 @@ func FilterPodByNamePrefix(pods *corev1.PodList, prefix string) *corev1.Pod {
 
 // GetService will get the service information for the input service name inside the input namespace
 func GetService(clientset *kubernetes.Clientset, namespace string, serviceName string) (*corev1.Service, error) {
-	return clientset.CoreV1().Services(namespace).Get(serviceName, metav1.GetOptions{})
+	svc, err := clientset.CoreV1().Services(namespace).Get(serviceName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
why we need this? already if there is any error, it will through the svc as nil right?

Reply:
It was returning an empty Service Object and not nil.

Reply:
Whenever we use this method, we will check for errors right? if there is no error, then only we will use the service right? Can we remove it? bcoz all the GetXXXX, ListXXXX is implemented in the same manner.

Reply:
I think, I have missed to check the error in GetService method. Can you pls add if !k8serror.IsNotFound, return error. and if err == nil, then patch the labels.

Reply:
In the above code we were not checking for an error. Also, this function will actually throw an error if the service does not exist. 
```
ERRO[0007] synopsyctl failed: failed to check for exposed service: services "alt-alert-exposed" not found
```

Reply:
Done!